### PR TITLE
Remove OpenCL tests

### PR DESCRIPTION
**Pull request by xinlipn**
Remove OpenCL test from cTest

### CODE REVIEW

**Comment by atamazov** [comment]
@xinlipn Is is ready to merge? If not, then please convert this PR to draft. Thanks.

**Comment by atamazov** [comment]
Why I am asking. This only removes regression test for issue 1012 (which is OCL specific indeed) but does not remove all OCL-specific tests. The PR should be either renamed or extended. Thanks.

**Comment by xinlipn** [comment]
> Why I am asking. This only removes regression test for issue 1012 (which is OCL specific indeed) but does not remove all OCL-specific tests. The PR should be either renamed or extended. Thanks.

Hi @atamazov , I searched by keywords OPENCL, HIP_DISABLED, OCL_ENABLED,  is_ocl_check, OCL_TEST_DEFAULT in test/CMakeLists.txt, this is the only test that shows up. 

Is there any other tests/logic you'd suggest to remove?

**Comment by xinlipn** [comment]
> @xinlipn Is is ready to merge? If not, then please convert this PR to draft. Thanks.

Please see my comment to the other question about more OPENCL tests. I can convert it to draft if you think there's more tests to be removed

**Comment by atamazov** [comment]
@xinlipn 
> Is there any other tests/logic you'd suggest to remove?

Yes. I'll better show this in my PR #2490. Stay tuned.

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -2275,27 +2275,6 @@ COMMAND	MIOPEN_FIND_MODE=normal MIOPEN_DEBUG_FIND_ONLY_SOLVER='GemmBwdRest' $<TA
...
-    MIOPEN_FIND_MODE=normal)
-
-set(ARGS_REGRESSION_ISSUE_1012
-    --verbose
-    --disable-forward
-    --disable-backward-data
-    --disable-validation)
-
-add_custom_test(test_regression_opencl_float_mi100 GFX900_DISABLED GFX906_DISABLED HIP_DISABLED GFX90A_DISABLED
-    # Issue #1012.
```
Comment (atamazov) [inline test/CMakeLists.txt:2289]: @xinlipn BTW issue #1012 is not OCL-specific and this change needs to be reverted.

**Comment by atamazov** [comment]
@xinlipn
> > Is there any other tests/logic you'd suggest to remove?
> 
> Yes. I'll better show this in my PR #2490. Stay tuned.

here it is https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2490/commits/5bf9244a932446c50f6ee3ffd5b476ce5ee353aa

**Comment by atamazov** [comment]
I think this can be closed

**Pull request closed without merging**